### ISSUE TITLE
setAttribute() error

### ISSUE THREAD

**Issue by ciadch** [Adldap2/Adldap2#103]
Hi @stevebauman !

Since your last update, I encouter a warning with setAttribute method : 

```
Warning: array_filter() expects parameter 1 to be array, string given
```

Why now we need to pass an array as first argument ?

Thanks for your help

**Comment by ciadch** [comment]
Mmmm, I found why ! 

Before the update, I change attributes like that : 
```php
$attributes = array('pwdlastset' => '0', 'lockouttime' => '0')
```
And now I need to change it like that : 
```php
$attributes = array('pwdlastset' => '0000', 'lockouttime' => '0000');
```

**Comment by stevebauman** [comment]
This is strange, going to investigate. Apologies for long response on this.

**Comment by stevebauman** [comment]
Added new `Adldap\Objects\BatchModification` object to handle model modifications. This has been fixed and will be released in `v5.2.0` coming shortly.
